roles/rhsm-subscription/tasks/main.yaml

```diff
@@ -53,12 +48,12 @@
         password: "{{ rhsm_password }}"
         state: present
         pool: "{{ rhsm_pool }}"
-      when: "'Current' not in subscribed.stdout and rhsm_user is defined and rhsm_user"
+      when: "'not registered' in subscribed.stdout and rhsm_user is defined and rhsm_user"
```

Comment:
Can we add an or in for 'Current' I know that @pschiffe needed something like this bc there are a few options in what the output is